target/linux/ramips/mt7621/base-files/lib/upgrade/platform.sh

```diff
@@ -96,6 +96,7 @@ platform_do_upgrade() {
 		iodata_mstc_upgrade_prepare "0x1fe75"
 		nand_do_upgrade "$1"
 		;;
+	iptime,a3004t|\
```

Comment:
Why `platform_upgrade_ubnt_erx`? I guess normal `nand_do_upgrade` will do the job?

Reply:
I just placed under of iotime. it's my mistake 🙈